lib/game.rb

```diff
@@ -28,8 +28,8 @@ def initialize(game_info)
     @season = game_info[:season]
     @type = game_info[:type]
     @date_time = game_info[:date_time]
-    @away_team_id = game_info[:away_team_id].to_i
-    @home_team_id = game_info[:home_team_id].to_i
+    @away_team_id = game_info[:away_team_id]
```

Comment:
Spec harness wants team_id as a string, so have to fix it here too so that they match